xbmc/video/dialogs/GUIDialogAudioSubtitleSettings.cpp

```diff
@@ -156,6 +158,16 @@ void CGUIDialogAudioSubtitleSettings::OnSettingChanged(const CSetting *setting)
       g_application.m_pPlayer->SetAudioStream(m_audioStream);    // Set the audio stream to the one selected
     }
   }
+  else if (settingId == SETTING_VIDEO_STREAM)
```

Comment:
On Blu-rays video streams can differ in the language used on screen, like for StarWars Blu-rays the famous scrolling intro text can be available in different languages, or on animated movies, they often translate stuff on objects in a scene (like signs etc).